plugins/com.google.cloud.tools.eclipse.usagetracker/src/com/google/cloud/tools/eclipse/usagetracker/AnalyticsPingManager.java

```diff
@@ -247,10 +244,13 @@ private void sendPing(PingEvent pingEvent) {
         }
       }
       
-      if (USE_CLEAR_CUT) {
+      if (useClearCut()) {
         try {
           String json = jsonEncode(pingEvent);
-          HttpUtil.sendPost(clearCutUrl, json, "application/json");
+          int resultCode = HttpUtil.sendPost(clearCutUrl, json, "application/json");
```

Comment:
So I noticed that _google-http-client_'s `HttpRequest#execute()`, which is used under the hood here, may throw an `HttpResponseException` for an error response when `HttpRequest#getThrowExceptionsOnExecuteError()` is true, which defaults to true.

Reply:
…which means this logging should be put below, I suspect.

Reply:
I think we're using java.net.URLConnection, not HttpRequest, here.

Reply:
No it's using google-http-client.

Reply:
I have to say it is not using Google HTTP Client. It is used only for the multi-part post that is not being used.

Reply:
Sorry I was looking at `sendPostMultipart` :blush: